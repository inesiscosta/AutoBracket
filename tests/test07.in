25 50
20 25 22 24 6 3 1 19 2 3 25 21 12 5 5 9 11 24 6 8 10 9 3 22 1
18 24 11 5 23 18 3 20 16 13 11 8 24 20 7 25 25 24 16 6 25 19 6 8 21
18 25 21 12 23 14 7 15 13 3 10 25 12 21 16 24 20 19 9 21 7 14 7 1 2
5 14 9 21 13 20 4 6 21 3 3 23 25 7 10 18 10 14 3 8 15 11 3 10 24
22 3 18 2 10 7 5 22 20 16 1 22 25 17 16 1 15 17 4 4 14 6 24 22 8
20 24 16 11 12 17 19 22 3 16 19 6 20 24 10 18 16 25 21 7 25 17 21 10 23
13 18 16 22 23 10 21 17 21 5 5 5 25 25 19 21 12 12 9 21 17 12 2 3 20
5 8 2 13 25 19 24 1 18 11 5 14 22 23 7 14 25 9 24 8 25 1 15 4 1
13 6 7 5 8 9 10 14 25 5 7 8 15 14 10 16 18 24 19 25 12 13 4 11 21
14 21 21 18 3 17 2 22 24 20 5 9 4 5 4 14 25 1 8 24 19 19 23 9 23
8 11 18 1 4 22 13 13 10 2 1 6 23 24 7 19 10 6 19 24 8 16 21 20 5
9 10 21 15 10 6 23 16 21 11 21 17 21 23 22 24 9 5 2 25 6 7 1 8 5
5 8 19 25 13 4 5 19 6 16 1 11 25 2 3 20 9 4 7 4 15 18 7 19 24
25 21 15 22 12 18 4 9 22 13 9 23 1 9 10 15 4 18 15 13 20 23 7 4 21
25 25 4 20 13 20 5 22 7 9 9 13 6 5 17 3 19 24 5 11 1 4 4 22 4
2 13 19 5 4 9 12 14 3 17 17 4 10 20 10 3 20 4 18 1 5 5 2 8 14
24 19 19 11 11 12 3 18 5 15 17 15 23 16 21 18 18 25 25 4 7 24 23 21 20
5 23 21 6 13 4 2 10 23 1 4 7 10 8 5 15 2 15 9 11 5 16 23 18 14
22 8 11 21 13 23 1 17 18 4 14 4 15 12 21 3 16 24 20 10 18 9 25 9 14
11 3 11 15 22 8 24 10 17 6 9 24 6 9 21 12 13 9 24 25 1 21 3 9 12
11 10 9 19 9 10 1 9 11 8 19 10 19 22 11 10 1 10 20 20 20 22 16 7 14
24 14 16 11 24 19 4 23 17 8 4 9 11 1 4 10 17 16 23 15 16 21 22 6 9
17 22 4 21 5 1 9 24 11 3 23 22 1 19 15 15 21 12 1 11 10 22 11 13 9
11 6 6 24 18 24 21 24 4 5 13 22 10 7 8 17 2 5 8 16 19 17 14 20 11
4 8 2 24 25 13 22 22 24 13 13 17 25 17 14 23 24 7 19 3 15 25 16 25 3
1 17 9 9 1 18 6 21 8 21 25 2 5 12 12 21 22 5 13 20 23 20 24 24 5 17 19 3 3 15 1 13 16 20 12 24 7 5 9 17 8 11 19 4 15 19 11 4 19 13
9
