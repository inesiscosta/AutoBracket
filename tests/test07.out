1
(((((((((((((((((((((((((((((((((((((((((((((1 17) (9 9)) 1) 18) 6) 21) 8) 21) 25) 2) 5) 12) 12) 21) 22) 5) 13) 20) 23) 20) 24) 24) 5) 17) 19) 3) 3) 15) (1 13)) 16) 20) 12) 24) 7) 5) 9) 17) 8) 11) 19) 4) 15) 19) 11) ((4 19) 13))
